staging/src/k8s.io/client-go/tools/clientcmd/api/types.go

```diff
@@ -30,53 +30,53 @@ type Config struct {
 	// Legacy field from pkg/api/types.go TypeMeta.
 	// TODO(jlowdermilk): remove this after eliminating downstream dependencies.
 	// +optional
-	Kind string `json:"kind,omitempty"`
```

Comment:
I tried doing this a while ago and I'm pretty sure it broke some of the `kubectl config` commands